lib/lhm/chunker.rb

```diff
@@ -16,7 +16,9 @@ class Chunker
     def initialize(migration, connection = nil, options = {})
       @migration = migration
       @connection = connection
-      @throttler = options[:throttler]
+      if @throttler = options[:throttler]
+        @throttler.connection = @connection if @throttler.respond_to?(:connection=)
```

Comment:
easy but is pretty bad design, I mean you are creating a useless object that then needs instantiation by hand.